x/auction/types/params.go

```diff
@@ -3,96 +3,81 @@ package types
 }
 
-// NewAuctionParams creates a new AuctionParams object
-func NewAuctionParams(maxAuctionDuration EndTime, bidDuration EndTime, startingID ID) AuctionParams {
-	return AuctionParams{
+// NewParams creates a new Params object
+func NewParams(maxAuctionDuration time.Duration, bidDuration time.Duration) Params {
+	return Params{
 		MaxAuctionDuration: maxAuctionDuration,
-		MaxBidDuration:        bidDuration,
-		StartingAuctionID:  startingID,
+		MaxBidDuration:     bidDuration,
 	}
 }
 
-// DefaultAuctionParams default parameters for auctions
-func DefaultAuctionParams() AuctionParams {
-	return NewAuctionParams(
+// DefaultParams default parameters for auctions
+func DefaultParams() Params {
+	return NewParams(
 		DefaultMaxAuctionDuration,
-		DefaultMaxBidDuration,
-		DefaultStartingAuctionID,
+		DefaultBidDuration,
 	)
 }
 
 // ParamKeyTable Key declaration for parameters
 func ParamKeyTable() subspace.KeyTable {
-	return subspace.NewKeyTable().RegisterParamSet(&AuctionParams{})
+	return subspace.NewKeyTable().RegisterParamSet(&Params{})
 }
 
 // ParamSetPairs implements the ParamSet interface and returns all the key/value pairs
 // pairs of auth module's parameters.
 // nolint
-func (ap *AuctionParams) ParamSetPairs() subspace.ParamSetPairs {
+func (ap *Params) ParamSetPairs() subspace.ParamSetPairs {
 	return subspace.ParamSetPairs{
 		{KeyAuctionBidDuration, &ap.MaxBidDuration},
 		{KeyAuctionDuration, &ap.MaxAuctionDuration},
-		{KeyAuctionStartingID, &ap.StartingAuctionID},
 	}
 }
 
-// Equal returns a boolean determining if two AuctionParams types are identical.
-func (ap AuctionParams) Equal(ap2 AuctionParams) bool {
+// Equal returns a boolean determining if two Params types are identical.
+func (ap Params) Equal(ap2 Params) bool {
 	bz1 := ModuleCdc.MustMarshalBinaryLengthPrefixed(&ap)
 	bz2 := ModuleCdc.MustMarshalBinaryLengthPrefixed(&ap2)
 	return bytes.Equal(bz1, bz2)
 }
 
 // String implements stringer interface
-func (ap AuctionParams) String() string {
+func (ap Params) String() string {
 	return fmt.Sprintf(`Auction Params:
 	Max Auction Duration: %s
-	Max Bid Duration: %s
-	Starting Auction ID: %v`, ap.MaxAuctionDuration, ap.MaxBidDuration, ap.StartingAuctionID)
+	Max Bid Duration: %s`, ap.MaxAuctionDuration, ap.MaxBidDuration)
 }
 
 // Validate checks that the parameters have valid values.
-func (ap AuctionParams) Validate() error {
-	if ap.MaxAuctionDuration <= EndTime(0) {
-		return fmt.Errorf("max auction duration should be positive, is %s", ap.MaxAuctionDuration)
-	}
-	if ap.MaxBidDuration <= EndTime(0) {
-		return fmt.Errorf("bid duration should be positive, is %s", ap.MaxBidDuration)
-	}
-	if ap.StartingAuctionID <= ID(0) {
-		return fmt.Errorf("starting auction ID should be positive, is %v", ap.StartingAuctionID)
-	}
+func (ap Params) Validate() error {
+	// TODO check durations are within acceptable limits, if needed
```

Comment:
Implement this. Main checks would be that bid duration and auction duration are positive, maybe that they aren't too small.